src/main/java/io/supertokens/storageLayer/StorageLayer.java

```diff
@@ -560,4 +565,131 @@ public static List<List<TenantIdentifier>> getTenantsWithUniqueUserPoolId(Main m
         }
         return result;
     }
+
```

Comment:
do we need functions like getUserMetadataStorage anymore in this file cause we can get it from tenantIdentifier / appIdentifier

Reply:
these should not be required any more.

Reply:
Will remove once all the changes are done, some tests still use it.

Reply:
ok. Keeping this comment open.

Reply:
Have added TODOs in the code to encourage this merge. Many tests use these function.